Gems/EMotionFX/Code/Source/Integration/Components/ActorComponent.cpp

```diff
@@ -155,6 +155,7 @@ namespace EMotionFX
                     ->Field("BoundingBoxConfig", &Configuration::m_bboxConfig)
                     ->Field("ForceJointsUpdateOOV", &Configuration::m_forceUpdateJointsOOV)
                     ->Field("RenderFlags", &Configuration::m_renderFlags)
+                    ->Field("ExcludeFromReflectionCubeMaps", &Configuration::m_excludeFromReflectionCubeMaps)
```

Comment:
can you bump the Version number on line 149 from 6 to 7 please?

Reply:
FWIW, when adding members, you do not need to bump the version number. Unless you know differently.